direct_chat.py

```
@@ -534,103 +518,112 @@ async def get_agent_status(
+):
+    """Resume a paused agent job with user input/action."""
+    session = _direct_chat_store.get(session_id)
+    if not session:
+        raise HTTPException(status_code=404, detail="Session not found")
+
+    # Store resume payload in session; the background job is polling for this
+    _direct_chat_store.update_resume_payload(session_id, req.model_dump())
+
+    # Also log the user's response in history for continuity
+    msg = f"Action: {req.action}"
+    if req.input: msg += f" - Input: {req.input}"
```

Ruff (0.15.13)

[error] Multiple statements on one line (colon)

(E701)